src/features/classification/security-controls/SecurityControlsItem.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../styles/variables';
+
+.bdl-SecurityControlsItem {
+    padding-left: 1em;
```

Comment:
em on purpose?

Reply:
Yes this is meant to keep the gap size for the bullet relative to the font size